### PR TITLE
267/import data in classroom

### PR DESCRIPTION
**Pull request by pyphilia**
In this PR: 
- factor out checkboxes of `LoadSelectionScreen` with `LoadSpaceSelectors`, in order to use them in `ImportDataAssignUserForm`. I also factor out load space related actions to reuse them.
- add space/actions/resources loading in a classroom functionality
- the imported data is not yet displayed in `StudentsTable` 

ref #267

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -59,9 +61,7 @@ import './App.css';
 import SavedSpaces from './components/SavedSpaces';
 import ClassroomScreen from './components/classrooms/ClassroomScreen';
 
-const styles = () => ({
-  toastrIcon: { marginBottom: '-20px', fontSize: '45px' },
-});
+const styles = () => ({});
```
Comment (pyphilia) [inline src/App.js:64]: This part shouldn't have been removed.

**Comment by pyphilia** [comment]
Some screenshots
image

image

image

image

If the wanted user does not exist, it can be created during the load in the select element.

File: src/components/classrooms/ImportDataScreen.js [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+                  className={classes.button}
+                >
+                  {t('Browse')}
+                </Button>
+                <Input
+                  required
+                  onChange={this.handleFileLocation}
+                  className={classes.input}
+                  inputProps={{
+                    'aria-label': 'Description',
```
Comment (juancarlosfarah) [inline src/components/classrooms/ImportDataScreen.js:189]: `t`

File: src/utils/syncSpace.js [hunk 3]
```diff
@@ -33,6 +33,20 @@ export const isSpaceUpToDate = (localSpace, remoteSpace) => {
   return _.isEqual(filteredLocalSpace, filteredRemoteSpace);
 };
 
+// return whether the localSpace is different from the remoteSpace
+// handle immutable spaces
+// @param localSpace : saved space in local database, immutable Map
```
Comment (juancarlosfarah) [inline src/utils/syncSpace.js:38]: Getting fancy with the documentation! For these function-level descriptions, I don't have a best practice yet. But often I see them wrapped in `/** */` multiline comments. I guess we can follow JSDoc, if you want.

https://gomakethings.com/whats-the-best-way-to-document-javascript/

**Pull request merged**